### ISSUE TITLE
Terminus : PingDb returns unknown promise ?

### ISSUE THREAD

**Issue by appsolutegeek** [nestjs/nest#2102]
<pre><code>
[ ] Regression 
[x] Bug report
[ ] Feature request
[x] Documentation issue or request
[ ] Support request => Please do not submit support request here, instead post your question on Stack Overflow.
</code></pre>

## Current behavior
I presume this isn't a bug but more of a missing documentation.

I am using the typeOrmHealthIndicators, and pingCheck seems to work and returns the correct promise.

But pingDb returns a promise of "unknown" so typescript is shouting at me here (pingDb)

```
      healthIndicators: [
        async () => this.typeOrmHealthIndicator.pingCheck('database', { timeout: 1500,connection: this.connection }),
        async () => this.typeOrmHealthIndicator.pingDb(this.connection)
      ],
```

What is the difference between pingCheck vs pingDb - not too sure.

Maybe some more examples for using the TypeOrmHealthIndicator could be given either in the repo or via the documentation.

## Environment

<pre><code>
Nest version: 6.1

 
For Tooling issues:
- Node version:11
- Platform:  MAC
Others:
</code></pre>

**Comment by BrunnerLivio** [comment]
This is related to [nestjs/terminus](https://github.com/nestjs/terminus) - please report it in the future in the corresponding repository.

Seems like I have forgotten to declare `pingDb` as a private method during my last change. Will fix asap. You should use `pingCheck`, as stated in the docs.

Thanks for reporting!

**Comment by appsolutegeek** [comment]
Ok, thanks, sorry :-) Wrong repo, I added some other additional issues in the same repo - promise I will do it over in the other repo - next time :-)

**Comment by BrunnerLivio** [comment]
Fixed with 6.1.4 :)

**Comment by appsolutegeek** [comment]
@BrunnerLivio Great!, Thanks
